src/devices/tuya.ts

```diff
@@ -2180,6 +2178,8 @@ const definitions: Definition[] = [
         },
         whiteLabel: [
             tuya.whitelabel('Lonsonho', 'X703A', '3 Gang switch with backlight', ['_TZ3000_rhkfbfcv']),
+            tuya.whitelabel('Zemismart', 'ZM-L03E-Z', '3 gang switch with neutral', ['_TZ3000_empogkya']),
+            tuya.whitelabel('TuYa', 'M10Z', '2 gang switch with 20A power socket', ['_TZ3000_lubfc1t5']),
```

Comment:
> 2 gang switch

This definition is for a 3 gang switch

Reply:
This device is a 2 Gang with a power socket that is also threated as a switch (same controls). Regardless, I will split them

Reply:
Do you think we need to split?

Reply:
Then this is fine!